config/samples/example-with-configmap.yaml

```diff
@@ -6,38 +6,44 @@ data:
   run.yaml: |
     # Llama Stack Configuration
     version: '2'
-    image_name: ollama
+    image_name: starter
     apis:
     - inference
     providers:
       inference:
-      - provider_id: ollama
+      - provider_id: ${env.ENABLE_OLLAMA:=__disabled__}
         provider_type: "remote::ollama"
         config:
           url: "http://ollama-server-service.ollama-dist.svc.cluster.local:11434"
     models:
-      - model_id: "llama3.2:1b"
+      - model_id: "ollama/llama3.2:3b"
         provider_id: ollama
         model_type: llm
+        provider_model_id: llama3.2:3b
+      - embedding_dimension: 384
+        model_id: ${env.ENABLE_OLLAMA:=__disabled__}/${env.OLLAMA_EMBEDDING_MODEL:=__disabled__}
+        provider_id: ${env.ENABLE_OLLAMA:=__disabled__}
+        provider_model_id: ${env.OLLAMA_EMBEDDING_MODEL:=__disabled__}
+        model_type: embedding
```

Comment:
Why do you need this extra entry ?

Reply:
Also, I think the LLS moved away from the `ENABLE_OLLAMA` "trick", and just uses `OLLAMA_URL`. See the `run.yaml` of the starter distribution.